### ISSUE TITLE
As a Linked Data user, I want to get the maximum use of information contained in the transcript field

### ISSUE THREAD

**Issue by mwigham** [beeldengeluid/beng-lod-server#19]
Recognised speakers, faces, places etc. are included in the transcripts as labels. These would be more usable if they were modelled as explicit metadata - now don't even know that these are people, locations etc. It would also be useful to split the transcripts up according to if they contain manual text or recognised entities.

**Comment by jblom** [comment]
@mwigham a bit too vague and quite far from must-have at this moment. Put it in the icebox, so we can work out the criteria better at a later time (if you agree)

**Comment by mwigham** [comment]
This issue related to the bg:transcripts metadata field. We will need to re-evaluate with the DAAN metadata to see if this issue is still relevant. It depends on how the enrichments (both old and new) are modelled in the logtrack items.

**Comment by jblom** [comment]
@mwigham I would rephrase this issue as:

as a data engineer, I want to model time based entities in the DAAN/B&G schema. 

I think it's much easier to understand what needs to be modelled; where these entities are from does not matter

**Comment by mwigham** [comment]
@jblom I agree, I've changed it. I have also updated the description to fit with the way the data is modelled in DAAN, as we will no longer be doing this for the iMMIx data.

**Comment by jblom** [comment]
@mwigham thanks, much clearer now!
